### ISSUE TITLE
Liberapay team thread - 2020

### ISSUE THREAD

**Issue by Changaco** [liberapay/salon#346]
Previous: #203. (Looks like no thread was created in 2019.)

**Comment by Changaco** [comment]
@blrhc Can you clarify why you've been taking €10 per week from the Liberapay team? I'm not sure what work you've been doing for Liberapay.

**Comment by Changaco** [comment]
@blrhc I've lowered your take to €1.

**Comment by Changaco** [comment]
@nijel I've set your take from the Liberapay team to €1 so that you don't capture all the leftover. Feel free to adjust your take to another amount, but please don't reset it to "auto".

By the way, it looks like Stripe is finally coming to your country (<https://stripe.com/global#CZ>).

**Comment by nijel** [comment]
@Changaco  Ah, I was not aware of `auto` doing this, sorry.

I'm on Stripe waiting list for years (I originally wanted to use it as payment gateway for Weblate), so let's see when it actually happens.

**Comment by MartinDelille** [comment]
After a few month of inactivity during which I removed my take to zero, I put it back to 1 €. My contribution are mostly issue comment but I'll try to end [my current pull requests](https://github.com/liberapay/liberapay.com/pulls/martindelille) in the next weeks! 😄

**Comment by Changaco** [comment]
@alvaroadlf Welcome to the team. Please set your take to a specific amount: https://liberapay.com/Liberapay/income/

**Comment by Changaco** [comment]
@alvaroadlf I've set your take to €1.

**Comment by Changaco** [comment]
@blrhc I've removed you from the team, since you've failed to respond to my clarification request and you've increased your take again.

**Comment by alvaroadlf** [comment]
> @alvaroadlf I've set your take to €1.

Perfect, sorry I didn't see the previous message  😔

**Comment by trebmuh** [comment]
@MuhdNurHidayat: thanks for providing the Malay translation for Liberapay up to 100%. This is awesome.

You went recently up to 50MYR which is roughly 10€ if I'm not wrong (please, correct me if so).
While it is perfectly understandable that you might deserve those, there is something a little embarrassing in the fact that several other translators (including me) take less than that for the same work in other languages (LP is fully translated in 11 languages, and 6 others languages are above 85% translated).

Unfortunately, there is not enough money yet  to cater for everyone needs it seems. Could you please then go back down to something more in line with others people take ? (As an example, I'm taking 5€/week for the French translation).

Best.

**Comment by jorgesumle** [comment]
Hmm... Maybe I'm too low then. I think I'm going to up it to 2€/week. I translate into Esperanto and Spanish (I get a lot of help with the Spanish translation though). I don't mind people taking more money if they are poorer, but I agree with @trebmuh that the difference between translators shouldn't be that big, and @MuhdNurHidayat shouldn't take more than 7€/week in my opinion.

**Comment by trebmuh** [comment]
@MuhdNurHidayat: can you make an answer please?

**Comment by trebmuh** [comment]
Well... I'm seeing right now that @MuhdNurHidayat have increased from 50MYR to 60MYR his/her take from the liberapay team... and didn't answer previous remark here which have been asked politely. I consider that this isn't very respectful for the team. @Changaco @jorgesumle and others obviously, what can be done now?

**Comment by jorgesumle** [comment]
Have you tried via email as well? If there is no response, then we should remove @MuhdNurHidayat from the team. We can later add them back when the communication is back to normal.

**Comment by alvaroadlf** [comment]
> Well... I'm seeing right now that @MuhdNurHidayat **have increased from 50MYR to 60MYR** his/her take from the liberapay team... **and didn't answer previous remark** here which have been asked politely.

 🤦🏻‍♂️ 🤦🏻‍♂️ 🤦🏻‍♂️ 🤦🏻‍♂️

**Comment by trebmuh** [comment]
@jorgesumle good idea. I've done so using the email address from his github account:

> Hi Muhammad,
>
>There is a question on the following team thread:
https://github.com/liberapay/salon/issues/346#issuecomment-610521867
>
>Can you please come there and talk with us?
>
>By advance, thank you.
>Cheers,
>Olivier

 Let see how it goes.

**Comment by mnh48** [comment]
oh sorry, I didn't realize people were discussing about me here... I have multiple projects piling up in github notification so I missed this...

thanks for the email @trebmuh 

the recent increase to MYR 60 was done (without me knowing the discussion going on here) to survive because I'm currently unemployed due to movement control order in place in my country

I don't receive any aids announced by my government because I'm considered as student in their database which made me inelligible for the single-adult aid despite already completed study, but at the same time I also didn't receive the student aid which is handled by university on behalf of government because, in university system it states I've already completed my study and no longer considered as student.

I'm reducing it to EUR 7 now (taking @jorgesumle as reference point for the max value) but do tell me if you want me to make it any lower and I'll look into other ways to survive.

Also my pronoun is they/them, I'm a nonbinary.

**Comment by trebmuh** [comment]
Thanks for answering and acting towards it. Yes, times are financially hard for a bunch of people those days. Being half under social financial aid and half under running a small one-person business, I'm also affected here and I'm sure other people too.

I think it could be great for anyone to communicate when changing the takes. It will avoid having doubt. I think that the "team thread" is a good place for that.

I'm then telling everyone here that I'm going from 5€ to 7€ too, given the fact we're on the same boat it seems.

**Comment by MartinDelille** [comment]
I put back my take to 0€ since my income are not yet (and might not be) affected by the current situation. I have a few time to dedicate to to project (except advertise it as bad as I do! 😓 ). Take care @MuhdNurHidayat !

**Comment by Changaco** [comment]
I have temporarily reduced my take by half so that more of any new payment will go to other members.

**Comment by MartinDelille** [comment]
@Changaco Why? In my opinion you still are doing most of the work here when looking at the contribution report: https://github.com/liberapay/liberapay.com/graphs/contributors

We should encourage the team member implicate themself instead of increasing the available take.

What about an open talk on https://meet.jit.si/ one day?

**Comment by Changaco** [comment]
The change has nothing to with anyone's contributions to the project, it's purely about routing incoming money to members who need it most in these difficult times. Me halving my take will result in more money from new payments being distributed to everyone else and especially @MuhdNurHidayat and @trebmuh.

**Comment by trebmuh** [comment]
That is very nice of you @Changaco . I've raised my take to 10€/week and sent an email to @MuhdNurHidayat to do the same.

**Comment by mnh48** [comment]
That is indeed very nice of you @Changaco .

Thanks for the email, @trebmuh .
I've also raised my take to 10€/week

**Comment by trebmuh** [comment]
@revi I saw you raised your take too. As proposed above, can you please communicate on it here next time you make a change of your take?

_Edit: revi said on IRC that he forgot to mention it here._

**Comment by trebmuh** [comment]
@ all : reminder that payday is tomorrow. Last few hours to set up / change your take if needed.

**Comment by Changaco** [comment]
[morvanc](https://liberapay.com/morvanc/) was capturing the entire leftover with his `auto` take, so I've changed it to €0.01.

**Comment by Changaco** [comment]
I've also changed all the other `auto` takes to zero to avoid future problems.

**Comment by Changaco** [comment]
I've increased my take back to €200.

The lowering of my take seems to have been somewhat successful, but the problem is that I have to restore my take every week before running Payday and lower it again afterwards, and this week I forgot to do it.

@mnh48 You should [connect a Stripe account](https://liberapay.com/about/me/payment) if you can, so that you'll receive more of the donations.

**Comment by mnh48** [comment]
thanks for your hard work

I have connected Stripe account.

The reason I didn't make an account when it first available in Malaysia was that it made business registration number a mandatory field to fill in, so it was impossible for me to create the account as I don't have any registered business in the first place.

Checked again today and the field is now optional, hence I'm now able to create an account there.
